integration/test-integration/src/test/resources/dockerCompose/backend-service-tls-with-opa-server.yaml

```diff
@@ -1,14 +1,19 @@
       - "5006:5006"
   mockBackend:
     image: wso2/choreo-connect-mock-backend:1.2.0-m1-SNAPSHOT
-    command: java -cp "lib/*" org.wso2.choreo.connect.mockbackend.MockServices -tls-enabled -interceptor-svc-enabled
+    command: java -cp "lib/*" org.wso2.choreo.connect.mockbackend.MockServices -tls-enabled -interceptor-svc-enabled -http2-tls-server-enabled
     ports:
       - "2370:2370"
       - "2371:2371" #Added this port for test debug purposes
       - "2383:2383"
       - "2399:2399"
     links:
       - router
+  mockBackend2:
+    image: wso2/choreo-connect-mock-backend:1.2.0-m1-SNAPSHOT
+    command: java -cp "lib/*" org.wso2.choreo.connect.mockbackend.MockServices -http2-server-enabled
+    links:
+      - router
```

Comment:
is there any reason why we are running a separate service?

Reply:
actually I could not get two http2 ports exposed from the same service as tests started to fail. Therefore, added a separate service